sbc_compassion/models/correspondence.py

```diff
@@ -76,6 +76,14 @@ class Correspondence(models.Model):
     child_id = fields.Many2one(
         related="sponsorship_id.child_id", store=True, readonly=False
     )
+    user_id = fields.Many2one(
+        "res.users",
+        string="Assigned to",
+        default=lambda self: self.env.user,
+        domain=[("share", "=", False)],
+        readonly=False,
+        required=True
+    )
```

Comment:
You should not add this field in the model, as we don't want to assign each letter individually to a user. A lot of letters imported automatically would be assigned to the user Administrator and therefore nobody will be notified when the letter is unsucessful.

Instead, you should add a setting in the sbc_settings.py class that will set a user for every letters, and it will be read from there when we assign the notification.